### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by OS-M**
December PR with editing README seems some kind of abandoned so i decided to make my own. 
I suggest replacing the `./target/release/near-indexer` with `./target/release/indexer-explorer` because is seems renamed in current version
And also as the https://github.com/nearprotocol/nearcore/issues/3156 is closed we dont need the NB! section anymore

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,17 +73,11 @@ $ cargo build --release
 To connect NEAR Indexer for Explorer to the specific chain you need to have necessary configs, you can generate it as follows:
 
 ```bash
-$ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
+$ ./target/release/indexer-explorer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
 ```
 
 The above code will download the official genesis config and generate necessary configs. You can replace `testnet` in the command above to different network ID (`betanet`, `mainnet`).
 
-**NB!** According to changes in `nearcore` config generation we don't fill all the necessary fields in the config file.
```
Comment (khorolets) [inline README.md:81]: While I agree with removing this paragraph, I would like to keep the links to configs as they might be useful for future reference.

**Review by khorolets — CHANGES_REQUESTED** [review]
Thank you for the PR! Small request from me

File: README.md [hunk 2]
```diff
@@ -73,17 +73,11 @@ $ cargo build --release
 To connect NEAR Indexer for Explorer to the specific chain you need to have necessary configs, you can generate it as follows:
 
 ```bash
-$ ./target/release/near-indexer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
+$ ./target/release/indexer-explorer --home-dir ~/.near/testnet init --chain-id testnet --download-config --download-genesis
```
Comment (khorolets) [inline README.md:76]: Nice catch, but I'm not sure. @frol what do you think maybe it's better to change the name in `Cargo.toml` instead?

File: README.md (hunk 1 above)
Comment (OS-M) [reply README.md:81]: Ok, but what is the way to keep them? Just keep the links or like "Here are the links to initial configs"?

File: README.md (hunk 2 above)
Comment (frol) [reply README.md:76]: @khorolets I would rename it in Cargo.toml to `near-indexer-for-explorer` to match the name of the repo and avoid confusion

Comment (OS-M) [reply README.md]: Ok, i discarder there changes. Should I do one more PR with changing `Cargo.toml`?

Comment (frol) [reply README.md]: @OS-M Yes, please, create a separate PR with the change in Cargo.toml and README

File: README.md (hunk 1 above)
Comment (frol) [reply README.md:81]: Yes, please, phrase it as these are the default config files that one could use just for the reference.

**Review by frol — APPROVED** [review]

**Comment by frol** [comment]
@OS-M Thanks! 🚀

**Pull request merged**